File: pronacej-backend-seguridad/src/main/java/net/latinus/sistema/integral/gestion/seguridad/model/both/PlanAsistenciaPostEgresoDTO.java
package net.latinus.sistema.integral.gestion.seguridad.model.both;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

@Data
@EqualsAndHashCode(callSuper = true)
public class PlanAsistenciaPostEgresoDTO extends CamposDTO implements Serializable {
    private Long idPlanAsistenciaPostEgreso;
    private String nombreEstado;
    private CatalogoDTO estado;
    private Date fechaInicio;
    private Date fechaFin;
    private String fecCreacion;
    private String fecInicio;
    private String fecFin;
    private List<PlanAsistenciaPostEgresoDetalleDTO> planDetalle;
    private String tokenFichaIdenticacion;
    private Long idFichaIdentificacion;
}

File: pronacej-front/src/app/core/model/both/planAsistenciaPostEgresoDTO.ts
import { CampoDTO } from "./campoDTO.model";
import { CatalogoDTO } from "./catalogoDTO.model";

export class PlanAsistenciaPostEgresoDTO extends CampoDTO {
  idPlanAsistenciaPostEgreso: number;
  nombreEstado: string;
  estado: CatalogoDTO;
  fechaInicio: Date;
  fechaFin: Date;
  fecInicio: string;
  fecFin: string;
  fecCreacion: string;
  planDetalle: PlanAsistenciaPostEgresoDetalleDTO[] = [];
  tokenFichaIdenticacion: string;
  idFichaIdentificacion: number;  
}

export class PlanAsistenciaPostEgresoDetalleDTO extends CampoDTO {
  idPlanAsistenciaPostEgresoDetalle: number;
  area: CatalogoDTO;
  factores: string;
  objetivoGeneral: string;
  objetivoEspecifico: string;
  actividades: string;
  institucion: string;
  frecuencia: string;
  indicador: string;
}